hhs_oauth_server/urls.py

```diff
@@ -46,6 +48,11 @@
     path("akamai/testobject", testobject, name="akamai_testobject"),
 ]
 
+if switch_is_active("v3_endpoints"):
+    urlpatterns += [
+        path("v3/fhir/", include("apps.fhir.bluebutton.v3.urls")),
```

Comment:
This seems not to work to enable/disable the v3 endpoints on the fly. I don't know much about this syntax, but what I see elsewhere is more like:
```
    path(
        "application-labels",
        waffle_switch("wellknown_applications")(ApplicationLabelView.as_view()),
        name="application-labels",
    ),
```
In any case I think we do need a way to be able to enable/disable these without a deployment, so hopefully you can play around with this and find something that works. I also could be doing something wrong, so if you're not seeing what I'm seeing, let me know.